appOPHD/Constants/Strings.h

```diff
@@ -262,7 +262,7 @@ namespace constants
 
 	const std::string AlertLanderLocation = "Lander Location";
 	const std::string AlertSeedTerrain = "The " + SeedLander + " cannot be placed on or near Impassable terrain.";
-	const std::string AlertSeedMine = "The " + SeedLander + " cannot be placed on or near a tile flagged with a Mine.";
+	const std::string AlertSeedOreDeposit = "The " + SeedLander + " cannot be placed on or near a tile flagged with a mine beacon.";
```

Comment:
I still tend to think of the markers as "mine beacons", though looking at this I'm wondering if we're being inconsistent here. Should we still call them "mine beacons"? At the game start, none of them will have any mines.